test/httpwstest.cpp

```diff
@@ -164,29 +158,15 @@ void HTTPWSTest::testSaveOnDisconnect()
                            socket1->waitForDisconnection(std::chrono::seconds(5)));
         LOK_ASSERT_MESSAGE("Expected successful disconnection of the WebSocket 2",
                            socket2->waitForDisconnection(std::chrono::seconds(5)));
-    }
-    catch (const Poco::Exception& exc)
-    {
-        LOK_ASSERT_FAIL(exc.displayText());
-    }
 
-    // Allow time to save and destroy before we connect again.
-    waitForKitProcessToStop(origPid,testname);
+        // Allow time to save and destroy before we connect again.
+        waitForKitPidsReady(testname);
 
-    TST_LOG("Loading again.");
-    try
-    {
+        TST_LOG("Loading again.");
         // Load the same document and check that the last changes (pasted text) is saved.
         std::shared_ptr<http::WebSocketSession> socket
             = loadDocAndGetSession(_socketPoll, _uri, documentURL, testname + "3 ");
 
-        // Should have no new instances.
-        std::set<pid_t> newPids = getDocKitPids();
-        LOK_ASSERT_EQUAL(static_cast<size_t>(1), newPids.size());
-        pid_t newPid = *(newPids.begin());
-        TST_LOG("New pid: " << newPid);
-        LOK_ASSERT_MESSAGE("New pid ("<<newPid<<") should not match old pid ("<<origPid<<")", newPid != origPid);
```

Comment:
Where did this go ?

Reply:
See the other comment

Reply:
Surely this checks that we don't create a new process for the same document when we're re-opening it (?)